dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -629,7 +629,7 @@ ssl.client.handshake.error.CWPKI0823E=CWPKI0823E: SSL HANDSHAKE FAILURE:  A sign
 ssl.client.handshake.error.CWPKI0823E.explanation=An error occurred during the SSL handshake.  It might require a signer export/import from the target host to the client TrustStore.
 ssl.client.handshake.error.CWPKI0823E.useraction=Review the extended error message from the TrustManager to determine what needs to change between the target SSL configuration and the client SSL configuration.
 # -------------------------------------------------------------------------------------------------
-ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host name verification error while connecting to host [{0}].  The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information.  The extended error message from the SSL handshake exception is: [{1}].
+ssl.client.handshake.error.CWPKI0824E=CWPKI0824E: SSL HANDSHAKE FAILURE:  Host name verification error while connecting to host [{0}].  The host name used to access the server does not match the server certificate''s SubjectDN or Subject Alternative Name information.  The extended error message from the SSL handshake exception is: [{1}].
```

Comment:
Change
"Host name verification error while connecting to host [{0}].  The host name used to access the server does not match the server certificate''s SubjectDN or Subject Alternative Name information.  The extended error message from the SSL handshake exception is: [{1}]."
to
Host name verification error while connecting to host [{0}]. The host name used to access the server does not match the server certificate's SubjectDN or Subject Alternative Name information. The extended error message from the SSL handshake exception is: [{1}]."